### PR TITLE
add support for q4os live

### PR DESCRIPTION
**Pull request by forkiesassds**

### CODE REVIEW

**Comment by thelamer** [comment]
This would also need the live.ipxe entry to link to it

**Comment by forkiesassds** [comment]
I added the live.ipxe entry.

File: src/live.ipxe [hunk 1]
```diff
@@ -9,6 +9,10 @@ item --gap Live Boot Distributions
 item live-debian ${space} Debian Live
 item live-fedora ${space} Fedora Live
 item live-ubuntu ${space} Ubuntu Live
+item live-kali ${space} Kali Live
+item live-tails ${space} Tails Live
+item žlive-elementary ${space} elementary OS Live
```
Comment (antonym) [inline src/live.ipxe]: Bad characters

**Review by antonym — CHANGES_REQUESTED** [review]
Bad characters

File: src/live.ipxe [hunk 2]
```diff
@@ -9,6 +9,10 @@ item --gap Live Boot Distributions
 item live-debian ${space} Debian Live
 item live-fedora ${space} Fedora Live
 item live-ubuntu ${space} Ubuntu Live
+item live-kali ${space} Kali Live
```
Comment (thelamer) [inline src/live.ipxe:12]: These items have been in alphabetical order should be like: 
live-debian
live-elementary
live-fedora
live-kali
live-q4os
live-tails
live-ubuntu

Also Please drop Tails, as is it should only be in development releases until we can figure out how to get networking working.

**Review by thelamer — CHANGES_REQUESTED** [review]

File: src/live.ipxe (hunk 1 above)
Comment (forkiesassds) [reply src/live.ipxe]: There fixed.

File: src/live.ipxe (hunk 2 above)
Comment (forkiesassds) [reply src/live.ipxe:12]: There. I have sorted them by alphabetical order, and dropped Tails.

**Review by thelamer — APPROVED** [review]

**Comment by antonym** [comment]
Pushing a generated dump from development up for testing here: https://github.com/antonym/netboot.xyz/pull/403

Closing this one out for that one, which should also cover Q4OS

**Pull request closed without merging**